LocalEngine.md

```diff
@@ -0,0 +1,35 @@
+# Running a Locally Built Flutter Engine
+
+It may be useful for debugging or development to run with a [locally built
+Flutter engine](https://github.com/flutter/flutter/wiki/Compiling-the-engine).
+To temporarily override the normal behavior of `update_flutter_engine` and
+`build_flutter_assets`, add a file called `engine_override` at the root of
```

Comment:
Just curious, why isn't this like .flutter_location_config? Passing the path to the engine as opposed to be forced to checkout the engine in a specific place?

Reply:
Just a benefit/effort tradeoff. `.flutter_location_config` allows for greater flexibility in the regular build environment, where there can be other constraints on the relative location of dependencies, whereas using `engine_override` is only meant for temporary local use (if a project wants to use a custom engine as a standard behavior, they just shouldn't use `update_flutter_engine` in their build for instance). So I think that where the engine checkout is located is much less likely to be an issue—especially since the Engine build docs themselves recommend using a folder called `engine` next to the Flutter tree.

Given that, writing the path configuration in Dart (making sure it handled relative paths, absolute paths, `~/...`, etc.) and testing all the variations didn't seem worth the investment of time at the moment.

If someone ends up needing that configurability, patches welcome at that point :)

Reply:
Makes sense, thanks!